### ISSUE TITLE
UX feature: Show both Nodes and Folders on map (remove selector)

### ISSUE THREAD

**Issue by willpopski** [meshifyiot/carbon-docs#237]
Since most customers will put location information on either Folders or Nodes, half of our users have to switch to either Folders or Nodes to see relevant locations on the map. This isn't great UX.

We should eliminate the selector box and show both Folders AND Nodes locations on the map.

**Comment by NateDawg90** [comment]
Isn't it necessary to distinguish to the user the difference between folders and nodes? A radio selector is a good way to do that. We could also add a "both" button to the selector.

**Comment by willpopski** [comment]
No, this isn't necessary. User with have either nodes or folders, rarely both, and they don't need to know the difference.

**Comment by NateDawg90** [comment]
Okay, going through with making an issue for this on applications.
